boards/avr-rss2/Makefile.dep

```diff
@@ -0,0 +1,6 @@
+USEMODULE += boards_common_atmega
+
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += bmx280
```

Comment:
```suggestion
  USEMODULE += bme280
```

`bme280` is a pseudo-module that selects the `bmx280` driver.
Also it looks like it doesn't have the driver-default I2C address of `0x77`, so you'll have to define `BMX280_PARAM_I2C_ADDR` in `board.h`

`i2c_scan` reports

```
2019-11-16 00:22:49,213 #  i2c_scan 0
2019-11-16 00:22:49,216 # Scanning I2C device 0...
2019-11-16 00:22:49,222 # addr not ack'ed = "-", addr ack'ed = "X", addr reserved = "R", error = "E"
2019-11-16 00:22:49,225 #      0 1 2 3 4 5 6 7 8 9 a b c d e f
2019-11-16 00:22:49,228 # 0x00 R R R R R R R R R R R R R R - -
2019-11-16 00:22:49,231 # 0x10 - - - - - - - - - - - - - - - -
2019-11-16 00:22:49,236 # 0x20 - - - - - - - - - - - - - - - -
2019-11-16 00:22:49,239 # 0x30 X - - - - - - - - - - - - - - -
2019-11-16 00:22:49,242 # 0x40 - - - - - - - - - - - - - - - -
2019-11-16 00:22:49,245 # 0x50 X - - - - - - - X - - - - - - -
2019-11-16 00:22:49,248 # 0x60 - - - - - - - - - - - - - - - -
2019-11-16 00:22:49,253 # 0x70 - - - - - - - - R R R R R R R R
```

Reply:
Using the 2.4 Rev board? Should on 77. I see with
tests/driver_bmx280:

Temperature [°C]: 24.0
Pressure [Pa]: 101301
Humidity [%rH]: 46.72

In Contiki avr-rss2 port
#define BME280_ADDR              (0x77 << 1) /* Alternative 0x76 */

BTW. There is a indoor AQ variant BME680. It can be soldered as a replacement for BME280. I did have sort of driver in Contiki. But the mapping to actual AQ was not knot known as that time.

Reply:
Ah I got the 2.3a revision of the board.

`0x58` is the EEPROM - what is `0x50` and `0x30`?

Reply:
AT24MAC has address selections pin. In avr-rss2 EUI chip get's 0xB0 and the EEPROM should get 0XA0 but I've never used the EEPROM actually you pointed out it was on another address. For 0x30 I'm confused. Where can I find the i2_scan your're using?

Reply:
> Where can I find the i2_scan your're using?

In e.g. `examples/default` add `USEMODULE += i2c_scan` to your `Makefile`.

Reply:
> 
> 0 1 2 3 4 5 6 7 8 9 a b c d e f
> 0x00 R R R R R R R R R R R R R R - -
> 0x10 - - - - - - - - - - - - - - - -
> 0x20 - - - - - - - - - - - - - - - -
> 0x30 X - - - - - - - - - - - - - - -
> 0x40 - - - - - - - - - - - - - - - -
> 0x50 X - - - - - - - X - - - - - - -
> 0x60 - - - - - - - - - - - - - - - -
> 0x70 - - - - - - - X R R R R R R R R    

> 
Ok. Right and shifted 0x50 & 0x58 for at24mac602 chip.
0x30 is puzzling. Hmm AtMegas can also be I2C slaves? TWAR register set?

Reply:
@benpicco You had another AtMega256RFR2 board? Can you run i2c_scan on this?

Reply:
I have a Microduino CoreRF with the ATmega128RFA1. It's basically just a breakout for the MCU with nothing connected but the bare necessities. 
There `i2c_scan` prints

```
2019-11-18 13:46:59,160 # Scanning I2C device 0...
2019-11-18 13:46:59,239 # addr not ack'ed = "-", addr ack'ed = "X", addr reserved = "R", error = "E"
2019-11-18 13:46:59,240 #      0 1 2 3 4 5 6 7 8 9 a b c d e f
2019-11-18 13:46:59,241 # 0x00 R R R R R R R R R R R R R R - -
2019-11-18 13:46:59,242 # 0x10 - - - - - - - - - - - - - - - -
2019-11-18 13:46:59,243 # 0x20 - - - - - - - - - - - - - - - -
2019-11-18 13:46:59,244 # 0x30 - - - - - - - - - - - - - - - -
2019-11-18 13:46:59,245 # 0x40 - - - - - - - - - - - - - - - -
2019-11-18 13:46:59,246 # 0x50 - - - - - - - - - - - - - - - -
2019-11-18 13:46:59,247 # 0x60 - - - - - - - - - - - - - - - -
2019-11-18 13:46:59,248 # 0x70 - - - - - - - - R R R R R R R R
```

Reply:
Thanks!
I desoldered the 24MAC602 chip 
> 0x00 R R R R R R R R R R R R R R - -
0x10 - - - - - - - - - - - - - - - -
0x20 - - - - - - - - - - - - - - - -
0x30 - - - - - - - - - - - - - - - -
0x40 - - - - - - - - - - - - - - - -
0x50 - - - - - - - - - - - - - - - -
0x60 - - - - - - - - - - - - - - - -
0x70 - - - - - - - - R R R R R R R R
> 
Same result. So it's the address chip. And reading the chip doc carefully on page 15. Chapter 11. Write protection:

> Once the permanent software write protection
> has been enabled, the device will no longer acknowledge the ‘0110’ (6h) control byte and cannot be reversed
> even if the device is powered down.

So 0x30 addr will dissapear when EEPROM becomes write protected. So I think we solved this...

Reply:
Alright, so what about the LEDs?
Are they wired different in version 2.3a?

Reply:
Yes LED has same wiring. Actually the rss2.h I was asked to remove has very concise summary of the boards.